examples/axum-example/Cargo.toml

```diff
@@ -70,7 +70,7 @@ site-pkg-dir = "pkg"
 # The assets-dir cannot have a sub directory with the same name/path as site-pkg-dir.
 #
 # Optional. Env: LEPTOS_ASSETS_DIR.
-assets-dir = "public"
```

Comment:
Not sure if Axum changed anything here, but it was complaining about this dir not existing when running cargo leptos serve. Since it's not used I assume we're good to just comment it out.